include/assimp/material.h

```diff
@@ -1548,7 +1548,7 @@ ASSIMP_API C_ENUM aiReturn aiGetMaterialIntegerArray(const C_STRUCT aiMaterial *
         int *pOut,
         unsigned int *pMax);
 
-#ifdef __cplusplus
```

Comment:
Same here, the old preprocessor directive is not active. Maybe you have introduced this for debugging proposes?

Reply:
No that is as intended, it is the fix -- not to use those macros, because they are broken as I explained.
The inline function however work fine in C (with the fix for the one missing C_STRUCT).

I kept the `#ifdef` in case the macros can eventually be fixed, but they are problematic with the material keys which gets expanded from 1 to 3 args (but _after_ the higher level macro invocation).